### PR TITLE
Notifications - Step 2

### PR DESCRIPTION
**Pull request by andreynering**
UI for notifications

### CODE REVIEW

File: cmd/web.go [hunk 1]
```diff
@@ -168,6 +168,8 @@ func runWeb(ctx *cli.Context) error {
 
 	bindIgnErr := binding.BindIgnErr
 
+	m.Use(user.GetNotificationCount)
```
Comment (bkcsoft) [inline cmd/web.go:171]: Doesn't this add unnecessary overhead for endpoints like `/api` that will _never_ use it? :)

File: public/less/_user.less [hunk 2]
```diff
@@ -74,4 +74,25 @@
...
+        .octicon {
+            float: left;
+            font-size: 2em;
+        }
+        .content {
+            float: left;
+            margin-left: 7px;
+        }
+
+        .octicon-issue-opened, .octicon-git-pull-request {
```
Comment (bkcsoft) [inline public/less/_user.less:88]: Aren't these defined somewhere already that we can reuse? (for the activity/dashboard feed)

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Otherwise 💯 🎉

File: cmd/web.go (hunk 1 above)
Comment (lunny) [reply cmd/web.go:171]: ^, Only UI route group need?

**Comment by metalmatze** [comment]
If you're creating UI changes a little screenshot would be pretty helpful for just browsing github! 👍 😊

File: public/less/_user.less (hunk 2 above)
Comment (andreynering) [reply public/less/_user.less:88]: I don't think so.

**Comment by andreynering** [comment]
Here a few pics:

- [1](https://cloud.githubusercontent.com/assets/7011819/21594733/371bc508-d10d-11e6-9264-9cd5a3ba6a83.png)
- [2](https://cloud.githubusercontent.com/assets/7011819/21594732/370f5cfa-d10d-11e6-8834-ab157169808f.png)
- [3](https://cloud.githubusercontent.com/assets/7011819/21594731/370a7604-d10d-11e6-90f9-9c8606233510.png)
- [4](https://cloud.githubusercontent.com/assets/7011819/21594730/36e99358-d10d-11e6-82d9-89efa83fd787.png)
- [5](https://cloud.githubusercontent.com/assets/7011819/21594729/36e69d56-d10d-11e6-8f92-ee547a76b3da.png)

It's working, but still very simple and missing some key functionality: pagination (must have), mark as read/unread, pinning a notification, others. Should this be done in this PR or on another ones?

File: cmd/web.go (hunk 1 above)
Comment (andreynering) [reply cmd/web.go:171]: Done

**Comment by bkcsoft** [comment]
Pagination is a must for this one, the others can be extended on in a later PR if necessary :)

**Comment by lunny** [comment]
^

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by strk** [comment]
@bkcsoft are your requested changes good now ?

**Comment by gsantner** [comment]
@andreynering
The Link in the UI is wrong when behind reverse proxy.
Mine is under `https://mydomain.xyz/gogitea` and everything links corretctly to subpaths (e.g. `https://mydomain.xyz/gogitea/issues`) but this one links to `https://mydomain.xyz/notifications` which, apparently doesn't exist.

**Comment by andreynering** [comment]
@gsantner Thank you for pointing. That's easy to fix.

**Pull request merged**